Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "raft"
-version = "0.4.0"
+version = "0.5.0"
```

Comment:
I think this should be "0.4.1"

Reply:
Ooops, this is an unintentional change. I'll fix soon

Reply:
Please fix it. :)

Reply:
Fixed.